Makefile

```diff
@@ -1,14 +1,35 @@
 TOPDIR          := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 include $(TOPDIR)/Makefile.env.mk
 
-DOCKER_DIRS      = agent topic-forwarder artemis broker-plugin api-server address-space-controller none-authservice standard-controller keycloak-plugin router router-metrics mqtt-gateway mqtt-lwt service-broker iot/iot-tenant-service iot/iot-auth-service iot/iot-device-registry iot/iot-http-adapter iot/iot-mqtt-adapter
-FULL_BUILD       = true
-
-GO_DIRS          = \
+GO_DIRS = \
 	enmasse-controller-manager \
 	iot/qdr-proxy-configurator \
 	iot/iot-gc \
 
+DOCKER_DIRS = \
+	agent \
+	topic-forwarder \
+	artemis \
+	broker-plugin \
+	api-server \
+	address-space-controller \
+	none-authservice \
+	standard-controller \
+	keycloak-plugin \
+	router \
+	router-metrics \
+	mqtt-gateway \
+	mqtt-lwt \
+	service-broker \
+	iot/iot-tenant-service \
+	iot/iot-auth-service \
+	iot/iot-device-registry \
+	iot/iot-http-adapter \
+	iot/iot-mqtt-adapter \
+
+FULL_BUILD       = true
```

Comment:
Shouldn't that be `?=` instead of `=`?

Reply:
I don't think it makes sense to make it configurable.  FULL_BUILD is basically the way to prevent the per module Makefile targets to trigger when running make targets in the toplevel Makefile (its ugly i know). See Makefile.java.mk, where its used to prevent mvn clean install to run for instance